pixart.js

```diff
@@ -1 +1,20 @@
+$(document).ready(function() {
+  $("button").on("click", colorBrush);
+  //input.val allows it to work for both click and return key for commits 1 and 2
+  function colorBrush(e) {
+    $(".brush").css("background", $("input").val());
```

Comment:
this is fine, but better might be

```js
var brushColor = $("input").val()
$(".brush").css("background",  brushColor);
```

obviously the same, but provides more clarity as to what your trying to do. Imagine other devs looking at your work, they won't really know which input your grabbing from and for what purpose. Semantics contributes significantly to code clarity.